File: Bubble Sort/Bubble Sort.py
# BUBBLE SORT IN PYTHON

# function algorithm that returns the sorted array
# algoritmo da funcao que retorna o array ordenado
def bubbleSort(arr):
	for i in range(len(arr) - 1):
		for j in range(i + 1, len(arr)):
			if arr[i] > arr[j]:
				swap = arr[i]
				arr[i] = arr[j]
				arr[j] = swap
	return arr

# code only to call and test the sort function
# codigo apenas para chamar e testar a funcao de ordenacao
arr = []

for i in range(10):
	arr.append(input())

arrSorted = bubbleSort(arr)

print("\nArray Sorted:")

for a in arrSorted:
	print(a)

File: Bubble Sort/Bubble Sort.rb
# BUBBLE SORT IN RUBY

# function algorithm that returns the sorted array
# algoritmo da função que retorna o array ordenado
def bubbleSort(arr)
	for	i in 0..arr.length - 2
		for j in i + 1..arr.length - 1
			if arr[i] > arr[j] then
				swap = arr[i]
				arr[i] = arr[j]
				arr[j] = swap
			end
		end
	end

	return arr
end

# code only to call and test the sort function
# codigo apenas para chamar e testar a funcao de ordenacao
arr = Array.new

for i in 0..9
	arr.push gets.to_i
end

arrSorted = bubbleSort(arr)

puts "\nArray Sorted:\n"
puts arrSorted;